Program.cs

```diff
@@ -39,13 +48,13 @@ public static List<Dog> GetDogs()
             blacky.Vaccination = new DateTime(2021, 10, 01);
             blacky.Deworming = new DateTime(2021, 07, 28);
             blacky.MedicalHistory = new List<Visit> ();
-            blacky.Clinic = animal;
+            blacky.ClinicName = listOfClinics[0].Name;
 
             Dog lucciano = new Dog("LUCCIANO", new DateTime(2015, 12, 24), "Lhasa Apso");
-            lucciano.Clinic = kaczor;
+            lucciano.ClinicName = listOfClinics[1].Name;
 
             Dog kaprys = new Dog("KAPRYS", new DateTime(2016, 02, 01), "Lhasa Apso");
-            kaprys.Clinic = kaczor;
+            kaprys.ClinicName = listOfClinics[1].Name;
 
             List<Dog> listOfDogs = new List<Dog>();

```

Comment:
You don't have to declare and initial again listOfDogs becouse You have it in line 21. Similarly with listOfClinics and list of cats, just use them here.